### PR TITLE
Multi-Label Text Classification and README Additions 

### PR DESCRIPTION
**Pull request by EricFillion**
- Completed the TC section of the README
- Added TC README examples to "examples/text-classification/readme-examples.py"
- Changed default model for TC to a non fine-tuned DISTILBERT model
- Added num-labels argument to HappyTextClassification 
- Added lots of new tests under "test/test_tc.py" to ensure multi-label predictions work on all three models 
- Added new testing data under "data/tc/test-multi.py" and "data/tc/train-eval-multi.py"
- Adding a config file with default values except for num_labels solved the "problem" of "distilbert-base-uncased-finetuned-sst-2-english" outputting "POSITIVE" and "NEGATIVE", while other models similar models were outputting "LABEL_0" and  "LABEL_1"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +143,191 @@ print(result[1].token_str)  # technology
 ## Text Classification 
 
 
+Initialize a HappyTextClassification() object to perform text classification.
```
Comment (ted537) [inline README.md:146]: `HappyTextClassification()` is a constructor; not a type

File: README.md [hunk 2]
```diff
@@ -143,6 +143,191 @@ print(result[1].token_str)  # technology
...
+
+This model assigns a label to a given text string. For example, you can train a model to 
+detect if an email is spam based on its text. 
+
+
+Initialization Arguments: 
+    1. model_type (string): either "ALBERT", "BERT" or "DISTILBERT." The default is "DISTILBERT"
+    2. model_name(string): below is a URL that contains potential models. The default is "distilbert-base-uncased"
+       [MODELS](https://huggingface.co/models?filter=text-classification)
+    3. num_labels(int): The number of text categories. The default is 2
```
Comment (ted537) [inline README.md:156]: Does not display well in github

File: README.md [hunk 3]
```diff
@@ -143,6 +143,191 @@ print(result[1].token_str)  # technology
...
+WARNING: If you try to load a pretrained model that has a different number of categories 
+than num_labels, then you will get an error 
+
+"albert-base-v2", "bert-base-uncased" and "distilbert-base-uncased" do not have a predefined 
+number of labels, so if you use these models you can set num_labels freely 
+
+### Initialization  
+
+"HappyTextClassification("ALBERT", "textattack/albert-base-v2-SST-2")"  has many useful applications. 
+It's able to detect the sentiment of text.
```
Comment (ted537) [inline README.md:167]: This statement implies that this ALBERT model is the only one that has these abilities. Seems confusing

File: README.md (hunk 2 above)
Comment (ted537) [reply README.md:156]: possibly related to the indentation

File: README.md [hunk 4]
```diff
@@ -143,6 +143,191 @@ print(result[1].token_str)  # technology
...
+ARGS_QA_TRAIN= {
+    'learning_rate': 5e-5,
+    'weight_decay': 0,
+    'adam_beta1': 0.9,
+    'adam_beta2': 0.999,
+    'adam_epsilon': 1e-8,
+    'max_grad_norm':  1.0,
+    'num_train_epochs': 3.0,
+
+}
```
Comment (ted537) [inline README.md:227]: Should be in code block

**Review by ted537 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (EricFillion) [reply README.md:225]: Fixed in d3b4534

File: README.md (hunk 3 above)
Comment (EricFillion) [reply README.md]: Fixed in d3b4534

File: README.md (hunk 2 above)
Comment (EricFillion) [reply README.md]: Fixed in d3b4534

File: README.md (hunk 1 above)
Comment (EricFillion) [reply README.md]: I changed it to "HappyTextClassification" in d3b4534

**Comment by EricFillion** [comment]
I applied your feedback to TC, WP and QA @ted537

**Review by ted537 — APPROVED** [review]
